### PR TITLE
Update curl command in the tunneling examples

### PR DESCRIPTION
**Pull request by vikaschoudhary16**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by ravenblackx — CHANGES_REQUESTED** [review]

File: configs/encapsulate_in_http1_connect.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # It can be used to test TCP tunneling as described in
 # https://envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/upgrades
 # and running `curl -x 127.0.0.1:10000 https://www.google.com`,
-# or running `curl -H "Host: the-hosted-domain-on-port-10003"  http://127.0.0.1:10002
+# or running `curl --resolve www.google.com:10000:127.0.0.1 https://www.google.com:10000`
```
[the remark this answers is not in the document]
Comment (vikaschoudhary16) [reply configs/encapsulate_in_http1_connect.yaml:10]: hey, I confirmed, it works as it is:
```
 $ curl --resolve www.google.com:10000:127.0.0.1 https://www.google.com:10000
<!doctype html><html itemscope="" itemtype="http://schema.org/WebPage" lang="en-IN"><head><meta content="text/html; charset=UTF-8" http-equiv="Content-Type"><meta content="/images/branding/googleg/1x/googleg_standard_color_128dp.png" itemprop="image"><title>Google</title><script nonce="TrsV5pWvKEfFCTTVWCZicw">(function(){var _g={kEI:'BGp-ZNzNEIDLseMPvcmFsA4',kEXPI:'0,793108,566301,6058,207,4804,2316,383,246,5,1129120,1704,1196003,380783,16115,28684,22431,1361,284,12028,4753,12834,4998,17075,38444,889,1983,2891,8349,3405,606,60690,6397,9938,1014,1,16916,2652,4,1528,2304,29062,13065,13658,4437,9358,13234,6645,7596,1,11943,30211,2,14022,2715,23024,5679,1020,31123,4567,6256,23419,1248,5841,14967,4333,8,7476,445,2,2,1,6960,19672,8155,7381,2,1478,14490,872,6578,13056,7,1922,9779,22887,19572,2007,18191,20137,14,82,2914,10419,6076,797,8377,3787,15201,3854,1521,3030,6110,5041,4665,1804,7734,2738,2884,11192,2738,5811,2171,8245,542,2365,2294,1110,1374,3317,2,2147,893,3579,1424,1143,7343,428,505,358,1105,1079,895,5349,1120,101,210,1454,3,724,2825,3,1076,818,528,2,4217,55,91,511,49,2423,1275,1267,696,643,227,92,44,265,1360,681,372,1178,577,21,1,1978,5200621,6230,48,136,329,94,49,84,2,5993727,2804424,3311,141,795,19736,1,346,6044,52,46,200,40,9,88,38,5,1,33,1,3,3,5,3,39,17,23943858,578,4041564,1964,16673,2893,6250,14715,4641,532,1395529,15379,2178384,21581568,85,93,134,1500,549,4,552,2,255,292,767,1405,1,201,510,451,680,2236,66,823,1619,728,167,1366,13,1280,774,215,352,127,351,219,68,80,843,9,331,496,203,364,5,996,6,175,1149,148,627,41,489,1222,22,162,1047,1,239,36,200,187,64,2,293,1,106,137,6,307,78,535,936,17,816,34,589,396,82,90,202,1,349,326,90,49,2,1,111,268,93,1,137,101,296,126,80,3,359,26,895,147,203,497,181,259,5,23,269,162,211,248,3,224,3,1131,370,225',kBL:'etGU',kOPI:89978449};if (!window.google || !window.google.stvsc){window.google = _g;}
})();(function(){google.sn='webhp';google.kHL='en-IN';})();(function(){
var h=this||self;function l(){return void 0!==window.google&&void 0!==window.google.kOPI&&0!==window.google.kOPI?window.google.kOPI:null};var m,n=[];function p(a){for(var b;a&&(!a.getAttribute||!(b=a.getAttribute("eid")));)a=a.parentNode;return b||m}function q(a){for(var b=null;a&&(!a.getAttribute||!(b=a.getAttribute("leid")));)a=a.parentNode;return b}function r(a){/^http:/i.test(a)&&"https:"===window.location.protocol&&(google.ml&&google.ml(Error("a"),!1,{src:a,glmm:1}),a="");return a}

```

File: configs/encapsulate_in_http2_connect.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # request which is sent upstream port 10001.
 # It can be used to test TCP tunneling as described in
 # https://envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/upgrades
-# and running `curl -x 127.0.0.1:10000 https://www.google.com`
+# and running `curl --resolve www.google.com:10000:127.0.0.1 https://www.google.com:1000`
```
[the remark this answers is not in the document]
Comment (vikaschoudhary16) [reply configs/encapsulate_in_http2_connect.yaml]: fixed. Thanks!

File: configs/encapsulate_in_http1_connect.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ravenblackx) [reply configs/encapsulate_in_http1_connect.yaml:10]: Ah, okay, maybe envoy splits the host header and removes the port from it. I'm not super familiar with that aspect.

Either way, this is still an improvement over the previous comment, so I'm okay with including or not including -H.

**Review by ravenblackx — APPROVED** [review]

File: configs/encapsulate_in_http1_connect.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (vikaschoudhary16) [reply configs/encapsulate_in_http1_connect.yaml:10]: :) yeah, I verified running a local upstream service and it does not work without passing `-H`, so your guess make sense that it is "google end of things".

**Review by ravenblackx — CHANGES_REQUESTED** [review]
Unapproving for addition of `-H` to examples. :)

**Review by ravenblackx — APPROVED** [review]

**Pull request merged**